src/Hazelcast.Net/Sql/SqlService.cs

```diff
@@ -90,7 +104,7 @@ public async Task<long> ExecuteCommandAsync(string sql, object[] parameters = nu
             var responseMessage = await _cluster.Messaging.SendAsync(requestMessage, cancellationToken).CfAwait();
             var response = SqlExecuteCodec.DecodeResponse(responseMessage);
 
-            if (response.Error != null) throw new HazelcastSqlException(response.Error);
+            if (response.Error != null) throw new HazelcastSqlException(_cluster.ClientId, response.Error);
```

Comment:
So, in this case, shouldn't we set the `clientId` to null or something to denote that the error is originated from the member, not the client?

Reply:
my idea was: the client is always known, so we always set the client. if member is known, then it comes from the member, and we also set the member. want to know if the error is originated from the member? then the member id is not Guid.Empty.

Reply:
Oh, I see :+1: